testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCIssuerEndpointTest.java

```diff
@@ -430,6 +438,57 @@ public void testCredentialIssuance() throws Exception {
     }
 
 
+    // Tests the AuthZCode complete flow without scope from
+    // 1. Get authorization code without scope specified by wallet
+    // 2. Using the code to get access token 
+    // 3. Get the credential configuration id from issuer metadata at .wellKnown
+    // 4. With the access token, get the credential
+    @Test
+    public void testCredentialIssuanceWithAuthZCode() throws Exception {
+ 
+        // 1. Get authoriZation code without scope specified by wallet
+        // 2. Using the code to get accesstoken
+        String token = getBearerToken(oauth.openid(false).scope(null));
+ 
+        // 3. Get the credential configuration id from issuer metadata at .wellKnown
+        Client client = AdminClientUtil.createResteasyClient();
```

Comment:
It might be better to use try-finally clause for this `Client clientForCredentialRequest`.
For example, see https://github.com/keycloak/keycloak/blob/8d1956ee5620b49eb2272559ad54f79cce615bd0/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/OIDCWellKnownProviderTest.java#L230-L245

Or, maybe, it can be re-written as follows:

```
        Response discoveryResponse;
        CredentialIssuer oid4vciIssuerConfig;
        try (Client client = AdminClientUtil.createResteasyClient()) {
            UriBuilder builder = UriBuilder.fromUri(OAuthClient.AUTH_SERVER_ROOT);
            URI oid4vciDiscoveryUri = RealmsResource.wellKnownProviderUrl(builder).build("test", OID4VCIssuerWellKnownProviderFactory.PROVIDER_ID);
            WebTarget oid4vciDiscoveryTarget = client.target(oid4vciDiscoveryUri);
            discoveryResponse = oid4vciDiscoveryTarget.request().get();
            oid4vciIssuerConfig = JsonSerialization.readValue(discoveryResponse.readEntity(String.class), CredentialIssuer.class);
        }
```